river/river.go

```diff
@@ -137,6 +139,22 @@ func (r *River) newRule(schema, table string) error {
 	return nil
 }
 
+func (r *River) updateRule(schema, table string) error {
+	rule, ok := r.rules[ruleKey(schema, table)]
+	if !ok {
+		return ErrRuleNotExist
```

Comment:
seem no need to care No Rule error.

Reply:
yes, it's unnecessary now, but I think that update something should return error when entity not found. so I add this.